src/js/yeldaWebchatInjector.js

```diff
@@ -792,6 +807,16 @@ class YeldaChat {
     this.webchatSettings = null
   }
 
+  /**
+   * Remove dom elements
+   * @param {String} id
+   */
+  removeElements(id) {
+    for (const element of document.querySelectorAll(`[id='${id}']`)) {
+      element.remove()
+    }
+  }
```

Comment:
```
/**
     * If init or setupChat has been called multiple times we might end up with multiple yelda_iframe_container and yelda_container
     * So to be sure that the destroy the webchat window completely, let's find all the matching elements and remove them all
     */
```
id are supposed to be unique in a DOM element => instead of deleting multiple element with the same id (which is not correct), can't we instead not create the duplicates ?
ex, when we init the chat, we could have check and do not init it if the yelda iframe already exist ?
what do you think ?
thanks

Reply:
code looks correct, i'll test it
concerning this feedback
1. do we still need `for (const element of document.querySelectorAll(`[id='${id}']`)) {` ? we should not have more than 1 element with the same id, right ?
2. if previous question answer is yet, we can also remove the comment about multiple yelda_iframe_container